### PR TITLE
Update fr_FR app translation for 3.8.2

### LINKED ISSUE #2303: Update fr_FR app translation for 3.8.2
Hi @jujudusud,
We are getting ready for the 3.8.2 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **fr_FR** by **2022-02-06**.

Please
- Update your fork from `jamulussoftware/jamulus` `master` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2022-02-06

Suggested commit and PR message:
```
Update fr_FR app translation for 3.8.2

Fixes #2303
```

Note: Some of the displayed "untranslated" strings will actually not be new, but will rather be shown due to technical reformatting (e.g. %1). You may be able to easily update them using the "vanished" translations list.

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/TRANSLATING.md).

Thanks for contributing to Jamulus!

### PR DESCRIPTION
**Pull request by jujudusud**
**Short description of changes**
Update fr_FR app translation for 3.8.2

**Context: Fixes an issue?**
Fixes #2303

**Does this change need documentation? What needs to be documented and how?**
No

**Status of this Pull Request**
Ready to be merged into master.

### CODE REVIEW

File: src/res/translation/translation_fr_FR.ts [hunk 1]
```diff
@@ -1565,42 +1565,42 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
...
         <location filename="../../clientsettingsdlg.cpp" line="189"/>
         <source>64 samples: Provides the lowest latency but does not work with all sound cards.</source>
-        <translation type="unfinished"></translation>
+        <translation>64 échantillons : Fournit la latence la plus faible mais ne fonctionne pas avec toutes les cartes son.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="195"/>
         <source>256 samples: Should only be used when 64 or 128 samples is causing issues.</source>
-        <translation type="unfinished"></translation>
+        <translation>256 échantillons : Devrait être utilisé uniquement si le réglage 64 ou 128 échantillons provoque des disfonctionnements.</translation>
```
Comment (hoffie) [inline src/res/translation/translation_fr_FR.ts:1593]:
```suggestion
        <translation>256 échantillons: Devrait être utilisé uniquement si le réglage 64 ou 128 échantillons provoque des disfonctionnements.</translation>
```
Redundant space before `:`?

**Review by hoffie — APPROVED** [review]
Looks fine to me, thanks! :)

- [x] Translator listed in the src/util.cpp, optionally add link to PR or code
- [x] Punctuation and spacing consistent.
- [x] Signal words consistent (“ASIO”, “Buffer”)
- [x] App translations: No untranslated strings (grep unfinished -5 src/res/translation/translation_$TRANSLATION*.ts)
- [x] App translations: Only a single .ts file checked in (.qm in addition is also OK)
- [x] Installer translations: Passes tools/check-wininstaller-translations.sh

File: src/res/translation/translation_fr_FR.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@
     <message>
         <location filename="../../util.cpp" line="392"/>
         <source>There is a server which collects  the audio data from each client, mixes the audio data and sends the mix  back to each client.</source>
-        <translation>Il y a un serveur qui collecte les données audio de chaque client, les mixe, et renvoie le mixage à chaque client.</translation>
+        <translation>Un serveur collecte les données audio de chaque client, les mixes et renvoie le résultat à chaque client.</translation>
```
Comment (trebmuh) [inline src/res/translation/translation_fr_FR.ts:29]: A la question "qui mixe les données ?", la réponse est "le serveur", c'est dont une 3ème personne de l'indicatif présent, donc "mixe" (pas de s final)

File: src/res/translation/translation_fr_FR.ts [hunk 3]
```diff
@@ -109,7 +109,7 @@
     <message>
         <location filename="../../util.cpp" line="546"/>
         <source>About %1</source>
-        <translation type="unfinished"></translation>
+        <translation>A propos %1</translation>
```
Comment (trebmuh) [inline src/res/translation/translation_fr_FR.ts:112]: À, pas A

File: src/res/translation/translation_fr_FR.ts [hunk 4]
```diff
@@ -761,7 +761,7 @@
     <message>
         <location filename="../../clientdlg.cpp" line="727"/>
         <source>%1 Directory</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 Annuaire</translation>
```
Comment (trebmuh) [inline src/res/translation/translation_fr_FR.ts:764]: Réflexion : ce n'est pas "directory" au sens de "répertoire" par hasard ?

**Review by trebmuh — CHANGES_REQUESTED** [review]
Pas eu le temps de tout relire, mais voici 3 commentaires.
Sinon, le reste me semble très bien, bravo.

**Review by ann0see — COMMENTED** [review]
Waiting until comments by trebmuh are resolved

File: src/res/translation/translation_fr_FR.ts (hunk 2 above)
Comment (jujudusud) [reply src/res/translation/translation_fr_FR.ts]: ok dans 965b2d3

File: src/res/translation/translation_fr_FR.ts (hunk 3 above)
Comment (jujudusud) [reply src/res/translation/translation_fr_FR.ts]: ok dans 965b2d3

File: src/res/translation/translation_fr_FR.ts (hunk 4 above)
Comment (jujudusud) [reply src/res/translation/translation_fr_FR.ts]: ok dans 965b2d3

**Comment by jujudusud** [comment]
> Waiting until comments by trebmuh are resolved

Done with 965b2d3

**Review by hoffie — APPROVED** [review]

**Review by ann0see — APPROVED** [review]

**Pull request merged**